installer/scripts/omsadmin.sh

```diff
@@ -813,6 +813,9 @@ copy_omsagent_d_conf()
 
     update_path $OMSAGENTD_DIR/monitor.conf
     update_path $OMSAGENTD_DIR/heartbeat.conf
+    if [ -s $OMSAGENTD_DIR/container.conf ] ; then
```

Comment:
I just did a file exists check, will change it to -f to be consistent with the rest of the file